src/main/java/com/microsoft/jenkins/artifactmanager/AzureArtifactConfig.java

```diff
@@ -112,5 +112,13 @@ public FormValidation doCheckContainer(@QueryParameter String container) {
             }
             return FormValidation.ok();
         }
+
+        public FormValidation doCheckPrefix(@QueryParameter String prefix) {
```

Comment:
I just wonder what's the reason about adding this function? I don't see the usage.

Reply:
It is used to check UI parameters. `doCheck**` will check parameter `**` when user enters it.